From a8cd9d288006c317d087bad9c4e0d2f399110184 Mon Sep 17 00:00:00 2001
From: Colin Guthrie <cguthrie@mandriva.org>
Date: Thu, 3 Mar 2011 09:37:16 +0000
Subject: [PATCH 6/7] conf: Make system.pa use udev-detect and not hal-detect.

Also fix a left over reference to HAL in default.pa
---
 src/daemon/default.pa.in |    2 +-
 src/daemon/system.pa.in  |    6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index 15e253f..1768c31 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -39,7 +39,7 @@ load-module module-card-restore
 load-module module-augment-properties
 
 ### Load audio drivers statically (it's probably better to not load
-### these drivers manually, but instead use module-hal-detect --
+### these drivers manually, but instead use module-udev-detect --
 ### see below -- for doing this automatically)
 #load-module module-alsa-sink
 #load-module module-alsa-source device=hw:1,0
diff --git a/src/daemon/system.pa.in b/src/daemon/system.pa.in
index 27e4281..4b1227f 100755
--- a/src/daemon/system.pa.in
+++ b/src/daemon/system.pa.in
@@ -20,11 +20,11 @@
 # mode.
 
 ### Automatically load driver modules depending on the hardware available
-.ifexists module-hal-detect@PA_SOEXT@
-load-module module-hal-detect
+.ifexists module-udev-detect@PA_SOEXT@
+load-module module-udev-detect
 .else
 ### Alternatively use the static hardware detection module (for systems that
-### lack HAL support)
+### lack udev support)
 load-module module-detect
 .endif
 
-- 
1.7.4.1

